bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2019A.xml

```diff
@@ -0,0 +1,169 @@
+<conf>
+    <smtp>smtp.hi.gemini.edu</smtp>
+
+    <bcc>swalker@gemini.edu</bcc>
+    <bcc>cquiroz@gemini.edu</bcc>
+    <bcc>sraaphorst@gemini.edu</bcc>
+
+    <type name="FastTurnaround">
+        <dir>/home/software/proposals/fast_turnaround/proposals/2018B</dir>
```

Comment:
I think the dir should be 2019A. Is this going to be used for testing?

Reply:
That was just me being stupid. Yes, it should be 2019A. Okay, pushed another commit for that.

Reply:
we should use `test_proposal` on the first stages